docs/modules/reference/pages/glossary.adoc

```diff
@@ -76,6 +91,9 @@ Provisioning:: The process of getting your devices, applications, and services i
 Remote Method Invocation (RMI):: Java API that lets one Java Virtual Machine (JVM) running object to invoke methods on an object running in another JVM. RMI integration lets you access Horizon/Meridian through jconsole or remote manage them.
 (See xref:operation:admin/rmi.adoc#rmi[Enabling RMI].)
 
+Requisitions:: Sets of nodes to be imported into Horizon for monitoring and management. Requisitions can be iteratively built and imported at a later date.
```

Comment:
```suggestion
Requisitions:: Sets of nodes to import into {page-component-title} for monitoring and management.
Requisitions can be iteratively built and imported at a later date.
```
The {page-component-title} attribute will substitute Horizon or Meridian depending on which is being built.